src/index.js

```diff
@@ -8,7 +8,8 @@ import style from './index.module.css';
 import SttTypeSelect from './select-stt-json-type';
 import ExportFormatSelect from './select-export-format';
 
-const tedTalkVideoUrl = 'https://download.ted.com/talks/KateDarling_2018S-950k.mp4';
+const tedTalkVideoUrl =
```

Comment:
Did the linter do this? it's not crazy long so I'd prefer it on one line

Reply:
yeah, must have been the linter, or prettier..